lib/pbench/common/logger.py

```diff
@@ -168,7 +168,7 @@ def get_pbench_logger(caller, config):
 
         handler.setLevel(logging.DEBUG)
         if config.log_fmt is None:
-            logfmt = "{asctime} {levelname} {process} {thread} {name}.{module} {funcName} {lineno} -- {message}"
+            logfmt = "{asctime} {levelname} {process} {thread} {name} {module} {funcName} {lineno} -- {message}"
```

Comment:
Could you try removing the parentheses in lines 160-162 and see if Blank provides a saner line wrapping?

Reply:
See PR #2309.

Reply:
Excellent exercise of discipline!